packages/compass-import-export/src/components/export-select-output/export-select-output.jsx

```diff
@@ -95,13 +94,13 @@ class ExportSelectOutput extends PureComponent {
             <Label htmlFor="select-export-file-output">Output</Label>
             <InputGroup bsClass={style('browse-group')}>
               <FormControl type="text" value={this.props.fileName} readOnly />
-              <IconTextButton
-                text="Browse"
+              <Button
                 id="select-export-file-output"
-                iconClassName="fa fa-folder-open-o"
-                clickHandler={this.handleChooseFile}
-                className={'btn btn-default btn-sm'}
-              />
+                leftGlyph={<Icon glyph="Save" />}
+                onClick={this.handleChooseFile}
+              >
+                Browse
+              </Button>
```

Comment:
Heads up: https://github.com/mongodb-js/compass/pull/3702 also touches this file, I’ve incorporated changes from this PR to avoid conflicts but it essentially undoes them